### DISCUSSION TITLE
Postgres batching

### DISCUSSION THREAD

**Discussion by satishviswanathan** [nhibernate/nhibernate-core#3261] [Ideas]
Any plans to add Postgres batching implementation similar to one available for Oracle?

**Comment by bahusoid** [comment]
It already supports batching (since v5.1+):

https://github.com/nhibernate/nhibernate-core/blob/d419ab3228f0edd3d56105851869f9e56f5d2bdc/src/NHibernate/Driver/NpgsqlDriver.cs#L93

Any issues?

**Reply by satishviswanathan** [reply]
Thank you for sharing the same. Any idea which version this feature was released ?

**Reply by bahusoid** [reply]
5.1
